src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -22,6 +22,8 @@ type BillingStatusResult = {
 function getBillingStatus(
     translate: <TKey extends TranslationPaths>(phraseKey: TKey, ...phraseParameters: PhraseParameters<Phrase<TKey>>) => string,
     cardEnding: string,
+    cardMonth: number,
```

Comment:
Wouldn’t it be better if we take one object cardInfo with all three params ? We could end up adding more arguments to this method later ?

Reply:
I was thinking about that but I decided not to. I doubt we will add more params. If you really don't like it I can change it :)

Reply:
I would recommend putting it as an object. Even if we don't have additional args added, all the params belong to one entity (and share common context).